client/lib/plans/test/get-popular-plan-type.js

```diff
@@ -12,14 +6,14 @@ import { TYPE_BUSINESS, TYPE_PERSONAL, TYPE_PREMIUM } from '../constants';
 
 describe( 'getPopularPlanType()', () => {
 	test( 'Should return TYPE_PERSONAL for personal site type', () => {
-		expect( getPopularPlanType( 'blog' ) ).to.equal( TYPE_PERSONAL );
+		expect( getPopularPlanType( 'blog' ) ).toEqual( TYPE_PERSONAL );
```

Comment:
This doesn't need to be changed, but I think `TYPE_PERSONAL` is a string. Typically we'd prefer `toBe` for strict comparison:

```suggestion
		expect( getPopularPlanType( 'blog' ) ).toBe( TYPE_PERSONAL );
```

Rule of thumb:
- Object/array/deep equality comparison: `toEqual`
- Literally same object comparison: `toBe`
- Other strict equality (booleans, strings, numbers): `toBe`

FYI: there are also special helpers for several of JavaScript's funnier values like `toBeUndefined()`, `toBeNull()`, and `toBeNaN()`.